src/commands/types.d.ts

```diff
@@ -24,6 +24,8 @@ type PatchedConfig = NetlifyTOML & {
   }
 }
 
+export type EnvironmentVariables = Record<string, { sources: string[], value: string; scopes?: string[] }>
```

Comment:
Might be nice to get more specific about the types here. How do you feel about something like:

```suggestion
type EnvironmentVariableScope = 'builds' | 'functions' | 'runtime' | 'post_processing'
type EnvironmentVariableSource = 'account' | 'addons' | 'configFile' | 'general' |  'internal' | 'ui'

export type EnvironmentVariables = Record<string, { sources: EnvironmentVariableSource[], value: string; scopes?: EnvironmentVariableScope[] }>
```

Reply:
Looks good!